### PR TITLE
minor change

### PR DESCRIPTION
**Pull request by lamngue**

### CODE REVIEW

File: src/containers/Order.js [hunk 1]
```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import DatePicker from "react-datepicker";
+import "react-datepicker/dist/react-datepicker.css";
+class Orders extends React.Component {
```
Comment (duyng404) [inline src/containers/Order.js]: @lamngue hey why did you add this file?

Comment (lamngue) [reply src/containers/Order.js]: e nham :<

**Review by duyng404 — APPROVED** [review]
